### PR TITLE
init jhipster module in app to have only one alert listener

### LINKED ISSUE #8498: Alert not working for user management with angular 6 frontend.
##### **Overview of the issue**

AlertService don't produce Alert on angular6 front end. 
<jhi-alert></jhi-alert> inside the template component is placed. 
in the http response, i create an header with the message to display inside the <jhi-alert></jhi-alert> component like that : 

ResponseEntity.created(new URI("/api/users/" + newUser.getLogin()))
		    .headers(HeaderUtil.createAlert("userManagement.created", newUser.getLogin())).body(utilisateurMappeur.toDTO(newUser));

Inside the notification interceptor service angular, the alert is created, but nothing is displayed inside the page. 

##### **JHipster Version(s)**

jhipster version 5.3.4

##### **JHipster configuration**

Using JHipster version installed locally in current project's node_modules
Executing jhipster:info
Options: from-cli: true
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
master-hipster-2@0.0.0 C:\dev\workspace\masterHipster2
`-- generator-jhipster@5.3.4

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "",
      "nativeLanguage": "fr"
    },
    "jhipsterVersion": "5.3.4",
    "applicationType": "monolith",
    "baseName": "",
    "packageName": "",
    "packageFolder": "",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "ehcache",
    "enableHibernateCache": true,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "postgresql",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "l",
    "clientFramework": "angularX",
    "useSass": true,
    "clientPackageManager": "npm",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "fr",
    "languages": [
      "fr",
      "en"
    ]
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

ls: no such file or directory: .jhipster/*.json
<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_101"
Java(TM) SE Runtime Environment (build 1.8.0_101-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.101-b13, mixed mode)

git version 2.7.1.windows.2

node: v8.11.3

npm: 6.4.1

yeoman: 2.0.2

yarn: 1.10.0

Congratulations, JHipster execution is complete!

##### **Browsers and Operating System**

All browser are impacted by this bug

### PR DESCRIPTION
**Pull request by atomfrede**
closes #8498

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/app.module.ts.ejs [hunk 1]
```diff
@@ -56,7 +57,14 @@ import {
         BrowserModule,
         <%=angularXAppName%>AppRoutingModule,
         Ng2Webstorage.forRoot({ prefix: '<%=jhiPrefixDashed %>', separator: '-'}),
-        <%=angularXAppName%>SharedModule,
+        NgJhipsterModule.forRoot({
+            // set below to true to make alerts look like toast
+            alertAsToast: false,
+            alertTimeout: 5000,
+            i18nEnabled: true,
```
[the remark this answers is not in the document]
Comment (atomfrede) [reply generators/client/templates/angular/src/main/webapp/app/app.module.ts.ejs]: Yes, you are right.

File: generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs [hunk 2]
```diff
@@ -28,15 +28,6 @@ import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
 @NgModule({
     imports: [
         NgbModule.forRoot(),
-        NgJhipsterModule.forRoot({
-            // set below to true to make alerts look like toast
-            alertAsToast: false,
-            alertTimeout: 5000,
```
[the remark this answers is not in the document]
Comment (atomfrede) [reply generators/client/templates/angular/src/main/webapp/app/shared/shared-libs.module.ts.ejs]: Ah interesting, was not aware of that. Will fix it.

**Comment by atomfrede** [comment]
@pascalgrimaud Do you have an idea why one of the azure builds is timing out while doing backend tests? Can see any connection between my changes and that test.

**Comment by pascalgrimaud** [comment]
@atomfrede : there was a timeout. I restarted the build

**Comment by atomfrede** [comment]
Thx. Lets see what happens.

**Comment by atomfrede** [comment]
It must be in the PR not the issue: Bounty claimed https://opencollective.com/generator-jhipster/expenses/5699

**Pull request merged**